pgcat.toml

```diff
@@ -105,23 +106,23 @@ statement_timeout = 15000
 [pools.sharded_db.shards.0]
 # [ host, port, role ]
 servers = [
-    [ "127.0.0.1", 5432, "primary" ],
-    [ "localhost", 5432, "replica" ]
+    [ "sharded-db-shard-0-primary", "127.0.0.1", 5432, "Primary" ],
+    [ "sharded-db-shard-0-replica-1", "localhost", 5432, "Replica" ]
```

Comment:
Do we really need to introduce explicit naming for instances? On one hand it is flexible but on the other hand it is error-prone, right?

Reply:
the name is to help with tagging the instances and helps to remove the need to identify based on index which is less descriptive than an explicit name